addons/dragon/$PBOPREFIX$

```diff
@@ -0,0 +1,2 @@
+z\ace\addons\dragon
+
```

Comment:
No empty line at end of pboprefix

Reply:
Did you just readd the empty line?

Reply:
I'm getting lots of conflicting messages and there is no style-guide anywhere to base my changes off of. I also couldn't find this request despite looking so I thought I just imagined it

Reply:
PBOPREFIX can have empty line at EOF, but not 2 of them. If I am seeing this correctly, it had 2, but right now it has none (which is also fine for PBOPREFIX, since it's always 1 line).